app/controllers/transactions_controller.rb

```diff
@@ -0,0 +1,36 @@
+class TransactionsController < ApplicationController
+
+  customer_tab :all
+  before_action :authenticate_user!
+  before_action :check_acting_as
+
+  include TransactionSearch
+
+  def initialize
+    @active_tab = "accounts"
+    super
+  end
+
+  def index_with_search
```

Comment:
I wonder if we should call this `/transactions/in_review` and reserve `/transactions` for the cross-account "my transactions" view we were talking about with NU today. Maybe not worth doing unless we actually do that feature.